### ISSUE TITLE
[BUG]Mouse Doesn't Work

### ISSUE THREAD

**Issue by JuiceTime360** [Amethyst-szs/smo-lunakit#50]
**Describe the bug**
The mouse is connected to the switch but the cursor won't move at all.

**To Reproduce**
Just plug in the mouse and for me the mouse won't work.

**Platform**
Nintendo Switch

**Comment by MrKatzenGaming** [comment]
This sounds like an issue on your end. Have you tried using the other usb ports of the switch?

**Comment by JuiceTime360** [comment]
Yes I tried all of the ports I even tried using a adapter to plug it directly into the switch.

**Comment by MrKatzenGaming** [comment]
The last thing I can think of is to try another mouse. If that doesn't work, I have no idea.

**Comment by JuiceTime360** [comment]
I tried but it didn't work.
